docs/getting-started/index.md

```diff
@@ -18,7 +18,7 @@ next:
 This tutorial will demonstrate the basic concepts behind [Terramate](https://github.com/mineiros-io/terramate) and give you some ideas for how a filesystem-oriented code generator can help you to manage your Terraform code at scale more efficiently .
 So that anyone can try Terramate without needing a cloud account, we will use only the `local_file` resource to create a static site demonstrating the basic principles behind Terramate. The only prerequisites are local installations of Terramate, Terraform and Git.
 
-> We launched a [Terramate Discord Server](https://terramate.io/discord)
+> We launched a [Terramate Discord Server](https://terramate.io/discord) in case you have any additional questions or issues running the eaples in this guide.
```

Comment:
```suggestion
> We launched a [Terramate Discord Server](https://terramate.io/discord) in case you have any additional questions or issues running the examples in this guide.
```